index.html

```diff
@@ -11,34 +11,217 @@
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#" class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#" class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#"class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                    </div>  
+                </div>
+            </div>
+        </div>
+        <section class="about-container">
+            <div class="first-about-container">
+                <h1 class="about-header">About me</h1>
```

Comment:
Convention and HTML best practices state that only one `h1 tag` should be used on a webpage. You have already used the `h1 tag` in the Headline section. Kindly change this tag to another `hx tag` between `h2-h6.` 💻